quill-core/src/main/scala/io/getquill/quotation/Parsing.scala

```diff
@@ -353,11 +353,15 @@ trait Parsing extends SchemaConfigParsing {
   private def is[T](tree: Tree)(implicit t: TypeTag[T]) =
     tree.tpe <:< t.tpe
 
+  private def isTraversable(tree: Tree) = {
```

Comment:
@getquill/maintainers This is not a strict check. Is there any better solution here?